Makefile

```diff
@@ -59,6 +65,16 @@ micro-iso: check $(PHOTON_STAGE) $(PHOTON_PACKAGES_MICRO)
                 -f > \
                 $(PHOTON_LOGS_DIR)/installer.log 2>&1
 
+micro-dep-list: check $(PREPARE_OUT_DATADIR)
```

Comment:
Replace with $(PHOTON_MICRO_DEPLIST) macro instead of using the value. For here and other places.